data/board-meeting.mail

```diff
@@ -1,11 +1,12 @@
 From: openSUSE Meeting Reminder <no-reply@opensuse.org>
 To: board@opensuse.org
-Subject: [reminder] Board Conf Call in two days at %(hour)s:%(minute)s NUE Time
+
+Subject: [reminder] Board Conf Call next Tuesday at %(hour)s:%(minute)s NUE Time
```

Comment:
The additional newline will move the subject line into the mail _body_, and the subject will be displayed as "(no subject)". Is this intentional?

Reply:
of course not....